### PR TITLE
fix compile in master

### PR DESCRIPTION
**Pull request by van800**

### CODE REVIEW

File: modules/mono/editor/GodotTools/GodotTools/BuildTab.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ private class BuildIssue : Reference // TODO Remove Reference once we have prope
         public bool ErrorsVisible { get; set; } = true;
         public bool WarningsVisible { get; set; } = true;
 
-        public Texture2D IconTexture
+        public Texture IconTexture
```
Comment (neikeq) [inline modules/mono/editor/GodotTools/GodotTools/BuildTab.cs]: Why is this needed? AFAIK `GetIcon` returns `Texture2D` and the only place `IconTexture` is used is being passed as a parameter that takes `Texture2D` as well. In fact, this change would break the latter.

**Comment by van800** [comment]
Sorry, my bad.

**Comment by neikeq** [comment]
BTW, I don't think C# bindings are working at all on latest master after recent breaking changes. I will fix them this week.
